modules/osdk-updating-128-to-131.adoc

```diff
@@ -0,0 +1,243 @@
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+----
+<1> Change the version from `{osdk_ver-n1}` to `{osdk_ver}`.
+
+. If you use a custom service account for deployment, define the following role to require a watch operation on your secrets resource as shown in the following example:
++
+.Example `<?>.yaml` file
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1 //is this correct?
+kind: ClusterRoleBinding
+metadata:
+  name: <operator_name>-admin
+subjects:
+- kind: ServiceAccount
+  name: <operator_name>
+  namespace: <operator_namespace>
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: ""
+rules: <1>
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - watch
+----
+<1> Add the `rules` stanza to create a watch operation for your secrets resource.
+endif::[]
+
+ifdef::ansible[]
+
+. Edit your Operator's Dockerfile to update the Ansible Operator version to {osdk_ver} as shown in the following example:
++
+.Example Dockerfile
+[source,docker]
+----
+FROM quay.io/operator-framework/ansible-operator:v{osdk_ver} <1>
+----
+<1> Update the Ansible Operator version from `{osdk_ver_n1}` to `{osdk_ver}`
++
+[WARNING]
+====
+The update to Ansible 2.15.0 in version 1.30.0 of the Ansible Operator removed the following pre-installed Python modules:
+
+* `ipaddress`
+* `openshift`
+* `jmespath`
+* `cryptography`
+* `oauthlib`
+
+If your Operator depends on one of these removed Python modules, update your Dockerfile to install the required modules using the `pip install` command.
+====
+
+. Edit your Operator project's makefile to update the Operator SDK version to {osdk_ver} as shown in the following example:
++
+.Example makefile
+[source,make]
+----
+# Set the Operator SDK version to use. By default, what is installed on the system is used.
+# This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
+OPERATOR_SDK_VERSION ?= v{osdk_ver} <1>
+----
+<1> Change the version from `{osdk_ver-n1}` to `{osdk_ver}`.
+
+. Edit your Operator's makefile to remove the `ansible-operator-2.11-preview` image as shown in the following example:
++
+[source,diff]
+----
+  # ...
+
+  # Convenience wrapper for building all remotely hosted images.
+  .PHONY: image-build
+- IMAGE_TARGET_LIST = operator-sdk helm-operator ansible-operator ansible-operator-2.11-preview scorecard-test scorecard-test-kuttl
++ IMAGE_TARGET_LIST = operator-sdk helm-operator ansible-operator scorecard-test scorecard-test-kuttl <1>
+  image-build: $(foreach i,$(IMAGE_TARGET_LIST),image/$(i)) ## Build all images.
+
+  # Convenience wrapper for building dependency base images.
+  .PHONY: image-build-base
+- IMAGE_BASE_TARGET_LIST = ansible-operator ansible-operator-2.11-preview
++ IMAGE_BASE_TARGET_LIST = ansible-operator <2>
+  image-build-base: $(foreach i,$(IMAGE_BASE_TARGET_LIST),image-base/$(i)) ## Build all images.
+  # ...
```

Comment:
This looks like an operator-sdk  makefile thing, not one scaffolded with ansible operators. I'd replace this with something to say that usage of the ansible-operator-2.11-preview base image should be replaced with the latest ansible operator base image if there isn't already a section for that (leaving these comments as I go).